filters/include/pcl/filters/impl/crop_hull.hpp

```diff
@@ -40,37 +40,25 @@
 
 #include <pcl/filters/crop_hull.h>
 
+
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-template<typename PointT> void
+template<typename PointT>
+PCL_DEPRECATED(1, 16, "This is a trivial call to base class method")
```

Comment:
@kunaltyagi Wasn't 1.13 correct? This is supposed to be a reminder that we remove this function just before releasing 1.13.0, right? Not a usual deprecation because the `applyFilter` from the `FilterIndices` will seamlessly replace this?

Reply:
🤦🏾 Yeah.
```suggestion
PCL_DEPRECATED(1, 13, "This is a trivial call to base class method")
```